sisyphus/cleaner.py

```diff
@@ -80,11 +85,17 @@ def extract_keep_values_from_graph():
     return job_dirs
 
 
-def find_too_low_keep_value(job_dirs, min_keep_value):
+def find_too_low_keep_value(
+    job_dirs: Union[str, Dict[Union[str, Path], int]],
+    min_keep_value: int,
+    filter_removed_jobs: Optional[List[Union[str, Path]]] = None,
+):
```

Comment:
now that you add typing, please also add the return type..

Reply:
that indeed is a good idea...